server/src/constants.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export const COMMIT_KIND = 'Commit';
+export const CUSTOMER_KIND = 'Customer';
+export const LISTING_KIND = 'Listing';
+export const MERCHANT_KIND = 'Merchant';
```

Comment:
@ifhuang regarding your comment in another PR [here](https://github.com/googleinterns/gpay-group-buy/pull/24#discussion_r439948152), I decided to change all the variable names like `commitsStorage` to `commitStorage` instead. Because it will standardise with the variables here. 

Here, kind names (table names in Datastore) use singular 'Commit' instead of 'Commits', so `COMMIT_KIND` makes more sense than `COMMITS_KIND`.

Another reason is that, usually it is said as 'merchant storage' not 'merchants storage', with the singular form for the noun in front.